docs/release-notes/NuGet-6.8.md

```diff
@@ -0,0 +1,211 @@
+* [marcin-krystianc](https://github.com/marcin-krystianc)
+  * [5293](https://github.com/NuGet/NuGet.Client/pull/5293) DetectAndMarkAmbiguousCentralTransitiveDependencies should be exhaustive and deterministic
+* [drewnoakes](https://github.com/drewnoakes)
+  * [5232](https://github.com/NuGet/NuGet.Client/pull/5232) Reduce allocations in RuntimeGraph
+* [Erarndt](https://github.com/Erarndt)
+  * [5218](https://github.com/NuGet/NuGet.Client/pull/5218) Reduce some allocations in CreateGraphNode.
+* [drewnoakes](https://github.com/drewnoakes)
+  * [5279](https://github.com/NuGet/NuGet.Client/pull/5279) Reduce allocations in VersionRangeFormatter
+* [drewnoakes](https://github.com/drewnoakes)
+  * [5248](https://github.com/NuGet/NuGet.Client/pull/5248) Reduce allocations in RuntimeDescription and RuntimeDependencySet
+* [drewnoakes](https://github.com/drewnoakes)
+  * [5269](https://github.com/NuGet/NuGet.Client/pull/5269) Don't box enumerators in ContentItemCollection
+* [drewnoakes](https://github.com/drewnoakes)
+  * [5250](https://github.com/NuGet/NuGet.Client/pull/5250) Don't allocate temporaries in FrameworkNameProvider.GetVersionString
+* [drewnoakes](https://github.com/drewnoakes)
+  * [5271](https://github.com/NuGet/NuGet.Client/pull/5271) Remove allocations from PackageSource.Source setter
+**TODO: Issues that could not be categorized. Make sure the issue has the correct milestone (if required) or an appropriate Type label - StillOpens:**
+
+* Badly specified framework leads to a non coded error in Visual Studio - [#12943](https://github.com/NuGet/Home/issues/12943)
+
+* Theme: Secure Supply Chain with NuGet - [#12502](https://github.com/NuGet/Home/issues/12502)
+
+* [Epic] Support SBOMs for NuGet packages - [#12497](https://github.com/NuGet/Home/issues/12497)
+
+* [Epic]: Add support for central package management to Visual Studio Package Manager UI - [#12231](https://github.com/NuGet/Home/issues/12231)
+
+* [ArPow] NuGet.Client should publish source-build intermediate packages - [#11059](https://github.com/NuGet/Home/issues/11059)
+
+* [Epic] NuGet Package Vulnerability Auditing - [#8087](https://github.com/NuGet/Home/issues/8087)
+
+* PM UI multi targeting experience is incomplete - [#4681](https://github.com/NuGet/Home/issues/4681)
+
+**TODO: Issues that could not be categorized. Make sure the issue has the correct milestone (if required) or an appropriate Type label - BreakingChanges:**
+
+* Remove the NuGetOperationType from NuGet.PackageManagement, use NuGetProjectActionType instead - [#12866](https://github.com/NuGet/Home/issues/12866)
+
+* Changing PackageVulnerabilityInfo severity from int to enum - [#12781](https://github.com/NuGet/Home/issues/12781)
+
+* Add nullable annotations to NuGet.Common - [#12775](https://github.com/NuGet/Home/issues/12775)
+
+* Obsolete Clone methods on immutable types - [#12669](https://github.com/NuGet/Home/issues/12669)

```

Comment:
All of these issues were already listed above in the breaking changes section near the top of the doc.

Reply:
nice catch, I'll take a look.